frontend/src/pages/articles/components/article-card/libs/components/popover-buttons-group/popover-buttons-group.tsx

```diff
@@ -0,0 +1,71 @@
+import { Button, Icon, Link } from '~/libs/components/components.js';
+import { AppRoute } from '~/libs/enums/enums.js';
+import { configureString } from '~/libs/helpers/helpers.js';
+import { useCallback } from '~/libs/hooks/hooks.js';
+import { type ArticleResponseDto } from '~/packages/articles/articles.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  isOwnArticle: boolean;
+  article: ArticleResponseDto;
+  onDeleteArticle: (id: number) => void;
+};
+
+const PopoverButtonsGroup: React.FC<Properties> = ({
+  isOwnArticle,
+  article,
+  onDeleteArticle,
+}) => {
+  const { id } = article;
+
+  const handleDeleteArticle = useCallback(() => {
+    onDeleteArticle(id);
+  }, [id, onDeleteArticle]);
+
+  return (
+    <div className={styles.buttonsGroup}>
+      <Button
+        label={
+          <>
+            <Icon iconName="favorite" />
+            <span>Add to favorites</span>
+          </>
+        }
+      />
+      {isOwnArticle && (
+        <>
+          <Link
+            to={
+              configureString(AppRoute.ARTICLES_EDIT_$ID, {
+                id: String(id),
+              }) as typeof AppRoute.ARTICLES_EDIT_$ID
+            }
+            state={article}
+            className={styles.editLink}
+          >
+            <Button
+              label={
+                <>
+                  <Icon iconName="edit" />
+                  <span>Edit</span>
+                </>
+              }
+            />
```

Comment:
I'm not sure you need this button inside link, when you can just style link same way as buttons in popover